src/Microsoft.NET.Sdk.Functions.MSBuild/Tasks/ZipDeployTask.cs

```diff
@@ -116,7 +116,7 @@ internal async System.Threading.Tasks.Task<bool> ZipDeployAsync(string zipToPubl
                     }
                     else if (status == DeployStatus.Failed || status == DeployStatus.Unknown)
                     {
-                        Log.LogError(String.Format(Resources.ZipDeploymentFailed, zipDeployPublishUrl, status));
+                        Log.LogError(String.Format(Resources.ZipDeployFailureErrorMessage, zipDeployPublishUrl, status));
```

Comment:
`ZipDeploymentFailed` is "Zip Deployment failed." [[source](https://github.com/Azure/azure-functions-vs-build-sdk/blob/v3.x/src/Microsoft.NET.Sdk.Functions.MSBuild/Properties/Resources.resx#L139)].
`ZipDeployFailureErrorMessage` is "The attempt to publish the ZIP file through {0} failed with HTTP status code {1}.`"[[source](https://github.com/Azure/azure-functions-vs-build-sdk/blob/v3.x/src/Microsoft.NET.Sdk.Functions.MSBuild/Properties/Resources.resx#L136)]